### DISCUSSION TITLE
README section on the web page follows system dark mode setting

### DISCUSSION THREAD

**Discussion by dan-sherwin** [verdaccio/verdaccio#4012] [Q&A]
I have installed verdaccio and all is working except one thing that is annoying.  I have dark mode set on my mac workstation.  I have set dark mode to FALSE in the verdaccio config.  I need that in light mode for my users.  When i bring up the we page and go to a package, the page itself is in light mode as it should be, but the readme file section is in dark mode.   When I change my system setting to light mode, then that section goes to light mode and vice versa.  Just the README section follows what my system is set to.  So now I have a light mode web page, and and dark mode readme file section, and it doesnt look good.  And of course if I click the setting to make the page dark mode, and turn my system to light mode, it is opposite as well.  I am attaching screen shots to show all four combinations.  Is there anyway to turn the dark mode off altogether across the whole site, and not have it follow the system settings?

<img width="1804" alt="Screenshot 2023-09-07 at 9 28 15 AM" src="https://github.com/verdaccio/verdaccio/assets/113303204/8c9c4b04-f05a-435a-8b8d-987ff17f6fcf">
<img width="1806" alt="Screenshot 2023-09-07 at 9 28 27 AM" src="https://github.com/verdaccio/verdaccio/assets/113303204/0de3ce70-9a15-41e5-be5f-abb6506d34db">
<img width="1807" alt="Screenshot 2023-09-07 at 9 28 01 AM" src="https://github.com/verdaccio/verdaccio/assets/113303204/9e5dbd9f-a755-4ff7-baa6-b270a0229e1e">
<img width="1809" alt="Screenshot 2023-09-07 at 9 27 45 AM" src="https://github.com/verdaccio/verdaccio/assets/113303204/a68cc531-3e77-449b-a8e6-0bf7e50bd9a8">

**Comment by juanpicado** [comment]
Thanks for reporting, is a bug already known , details here https://github.com/verdaccio/verdaccio/discussions/3942

**Comment by dan-sherwin** [comment]
Dang it.  Sorry for the dupe.  I looked but missed that one.
